engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -328,7 +328,7 @@ public void setVignette(boolean vignette) {
     }
 
     public boolean isMotionBlur() {
-        return motionBlur && !isVrSupport();
+        return motionBlur;
```

Comment:
Sorry, I just read a comment from you on this topic via email. Disregard the comment above. Indeed if some effects are incompatible with VR enabling VR should disable them automatically and perhaps leave the choice to the user to re-enable them to see what happens - this would also enable us developers to assess the problems and potentially solve them.